tslearn/metrics.py

```diff
@@ -12,14 +12,77 @@
 from sklearn.utils import check_random_state
 from tslearn.soft_dtw_fast import _soft_dtw, _soft_dtw_grad, \
     _jacobian_product_sq_euc
+from tslearn.cysax import cydist_sax
 
 from tslearn.utils import to_time_series, to_time_series_dataset, ts_size, \
     check_equal_size
 
 __author__ = 'Romain Tavenard romain.tavenard[at]univ-rennes2.fr'
 
 GLOBAL_CONSTRAINT_CODE = {None: 0, "": 0, "itakura": 1, "sakoe_chiba": 2}
-VARIABLE_LENGTH_METRICS = ["dtw", "gak", "softdtw"]
+VARIABLE_LENGTH_METRICS = ["dtw", "gak", "softdtw", "sax"]
+
+
+def cdist_sax(dataset1, breakpoints_avg, size_fitted, dataset2=None,
```

Comment:
The reason why you cannot @njit this function is probably because it relies on Cython code. This code could probably be ported to numba at some point, but this is not the point of this PR, I would say.